index.html

```diff
@@ -13,23 +13,202 @@
-				<li><a href="#"><img src="./images/github.svg" alt="Github Logo"></a></li>
-				<li><a href="#"><img src="./images/angellist.svg" alt="Angellist Logo"></a></li>
-			</ul>
-		</main>
+
+		<div class="app-items">
+			<nav class="app-nav">
+				<a id="logo" href="#">My logo</a>
+				<i id="menu-icon" class="bi bi-list"></i>
+			</nav>
+			<main class="app-greeting">
+				<h1>I'm Sane great to see you!</h1>
+				<p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+					and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+				<p id="lets-connect">LET'S CONNECT</p>
+				<ul class="socials">
+					<li><a href="#"><img src="./images/twitter.svg" alt="Twitter logo"></a></li>
+					<li><a href="#"><img src="./images/linkedin.svg" alt="Linkedin Logo"></a></li>
+					<li><a href="#"><img src="./images/medium.svg" alt="Medium Logo"></a></li>
+					<li><a href="#"><img src="./images/github.svg" alt="Github Logo"></a></li>
+					<li><a href="#"><img src="./images/angellist.svg" alt="Angellist Logo"></a></li>
+				</ul>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/tonic1-img.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Tonic</h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="projects">
+			<div class="project-images">
+				<img src="./images/availability.svg" alt="Picture of project">
+			</div>
+			<main class="project-title">
+				<h2>Multi-Post<br> Stories</h2>
+				<ul class="list">
+					<li id="canopy">Canopy</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">Back End Dev</li>
+					<li><img src="./images/Counter.svg" alt=""></li>
+					<li id="backend">2015</li>
+				</ul>
+				<p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.
+				</p>
+				<ul class="program-languages">
+					<li><a id="language-name" href="#">html</a></li>
+					<li><a id="language-name" href="#">css</a></li>
+					<li><a id="language-name" href="#">javascript</a></li>
+				</ul>
+				<div><a class="see-projects" href="#">See Projects</a></div>
+			</main>
+		</div>
+
+		<div class="projects">
```

Comment:
- [ ] Please make sure that each project card has the same spacing `padding/margin` left right as defined in the Figma template and also your button and spacing between elements need t to be the same. It is important to stick to the template as much as possible :+1: 

#### Figma template
![Screenshot from 2022-09-28 11-15-22](https://user-images.githubusercontent.com/6615394/192766237-8f3f4d05-348c-4ea1-ad45-812e61bdebcd.png)

#### Your design
![Screenshot from 2022-09-28 11-13-37](https://user-images.githubusercontent.com/6615394/192766242-890814f4-e754-4e19-9a75-c1b22e53da88.jpg)